### DISCUSSION TITLE
how to set default theme to Dark

### DISCUSSION THREAD

**Discussion by phpiscute** [vuepress/core#564] [Q&A] [answered]
Dark theme is cute, how to set it to default

**Comment by hschaefer123** [comment]
did you try
themeConfig: {
    darkMode: true,
    ...
}

BR
Holger

**Reply by phpiscute** [reply]
Which version support it?
thx  Holger~

**Comment by hschaefer123** [comment, marked as answer]
I think i used it it 2.0.0-beta26 or 27 but setting it to white or black only to disable selector.
Maybe they removed the option.
I am also having issues with vite, so i need to dig into more details.

Just had a look into the default theme
https://github.com/vuepress/vuepress-next/blob/main/packages/%40vuepress/theme-default/src/shared/options.ts

The option is still there, but it seems to be only for the toggle and to avoid darkMode. Maybe feature hardly depends on used style in os?

There is also
vuepress-color-scheme', 'auto'
maybe it can be set to dark.

Maybe you can make a feature request, because currently it seems to be only possible by config to disable darkMode toggle and avoid darkMode. 

BR
Holger

**Reply by phpiscute** [reply]
very thanks for Holger~~
